### ISSUE TITLE
Logged training loss does not decrease while performance improves

### ISSUE THREAD

**Issue by jumpyknight** [google/deepvariant#194]
I'm trying to train a deepvariant model with a very simple topology.
After a few thousands of training steps, the logged training loss starts to vibrate around a rather high value. However the performance of saved models still keeps improving on my validation data set.
Why does this happen?

**Comment by sidharthgoel** [comment]
Hi, can you share the commands you ran and the associated logs so we can take a look?

**Comment by pichuan** [comment]
If you're plotting on TensorBoard, it'll also be helpful to share a screenshot.

**Comment by jumpyknight** [comment]
![train_loss_and_error](https://user-images.githubusercontent.com/13111474/60637174-f2844000-9e4b-11e9-9ed0-6461041cf407.png)
Hi, I looked in my log file and plotted a figure of training losses and validation f-measures, and it turned out that I might have overlooked the improves in the trained models. The improvments after 10000 steps become very small (still important though) compared with that in the first steps.

**Comment by sidharthgoel** [comment]
It's normal for the loss curve to start flattening out, although yours does change pretty abruptly. What exactly did you change (you mentioned using a `very simple topology`, not sure if that means you changed the architecture or something). Also the commands you used would be helpful.

**Comment by jumpyknight** [comment]
Thanks for the comments. I tried to train a simple 10-cnn-layer architecture and got the above results. Although the final loss still seems to be rather high, the accuracy is acceptable for the simple cnn network. I will close this issue.
